docs/04_For_Developers/06_Project_Maintainer_Guide.md

```diff
@@ -0,0 +1,58 @@
+# Project Maintainer Guide
+RSS-Bridge project maintainers are responsible for:
+1. Tagging issues
+1. Reviewing and merging pull requests
+1. Creating releases
+
+Project maintainers should keep all communications related to the project open
+to the community (Preferably GitHub issues but IRC/Matrix is also allowed).
+
+## Issues
+Maintainers should triage and respond to issues if possible, making sure to
+specify the tag if possible. For example, the 'Bug Report' can be clarified to
+'Bridge-Broken' once that is established. Project maintainers are encouraged to
+mention the bridge maintainer to comment on issues related to a specific bridge.
+
+An issue should remain open until it is resolved by fixing the issue or adding
+the feature it describes. There are some exceptions:
+- In the case of a feature request that will not be added, the issue can be
+  closed with the 'wontfix' label after pinging other active maintainers.
+- If an issue is too obscure or unclear and the issue creator does not respond
+  to additional clarification questions for 1 month, these issues should be
+  closed as they cannot be acted upon.
+
+## Pull Requests
+Maintainers should thoroughly **review** and **test** pull requests before
+merging them. Most bridge authors use other bridges in the project for reference
+so it is important to maintain a high quality codebase so that it does not
+deteriorate.
+
+To merge a pull request, use the "Squash and Merge" option in GitHub to compress
+the commits to a single change. When doing this, remove the extra commit
+description. However, if multiple authors contribute to a pull request, do not
+remove the Co-authored-by lines at the end of the message.
+
+Maintainers must not push to master, but create pull requests instead.
```

Comment:
I think it's silly to not allow maintainers to push to master. It's been discussed in IRC so I won't repeat the arguments here.

Reply:
Yes, this topic has already been resolved.